### PR TITLE
Add a Jenkins step for alerting changes to homepage

### PR DESCRIPTION
**Pull request by ncksllvn**
This PR adds a Jenkins step so that when a PR is opened that modifies the homepage, we receive a Slack notification in our oncall channel.

image

Ticket - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16883

### CODE REVIEW

**Comment by nick** [comment]
@va-bot keeps paging me :-/

**Comment by ncksllvn** [comment]
Sorry @nick, turning that off now...

File: Jenkinsfile [hunk 1]
```diff
@@ -43,11 +51,32 @@ node('vetsgov-general-purpose') {
...
+Pull request opened containing changes to the VA.gov homepage! \
+These changes usually contain content that is high priority, such as for a weather alert or government shutdown. \
+Please review, merge, and if necessary, deploy this change as soon as possible. \
+https://www.github.com/${GITHUB_ORG}/${CONTENT_REPO}/pulls
+"""
+
+        slackSend(message: message, channel: 'oncall', color: '#DDDD00', failOnError: false)
+      }
     }
+  }
```
Comment (ncksllvn) [inline Jenkinsfile]: I wanted to link to the actual PR, but every time I tried [`getNumber`](http://github-api.kohsuke.org/apidocs/index.html) an error was thrown in Jenkins.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1 @@
-/fragments/home/ @ncksllvn @jbalboni @1Copenut
```
Comment (ncksllvn) [inline .github/CODEOWNERS]: This is no longer needed, now that the `oncall` channel will be pinged.

**Comment by ncksllvn** [comment]
@wyattwalter @tskinn If you have time, can you review this PR? I'd like to demo it tomorrow.

....I'm fighting off the temptation to merge this using admin privileges 😱 🔥 🔨 🚀

**Pull request merged**